examples/src/main/scala/org/apache/spark/examples/h2o/AirlinesWithWeatherDemo.scala

```diff
@@ -35,6 +35,7 @@ object AirlinesWithWeatherDemo extends SparkContextSupport {
     // Configure this application
     val conf: SparkConf = configure("Sparkling Water: Join of Airlines with Weather Data")
 
+    FIXME replace with spark Session
```

Comment:
Thx, Kuba, i preserve separation of individual steps - you have to look at final diff.